microshift_support/microshift-etcd.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-The etcd service is delivered as part of the {product-title} RPM. The etcd service is run as a separate process and the etcd lifecycle is managed automatically by {microshift-short}.
+The {product-title} RPM includes the etcd service. Etcd runs as a separate process and {microshift-short} automatically manages the etcd lifecycle.
```

Comment:
For a future PR: "etcd" is always lowercase, even at the beginning of a sentence.